uncolorize.js

```diff
@@ -12,7 +12,7 @@ const { MESSAGE } = require('triple-beam');
  */
 module.exports = format((info, opts) => {
   if (opts.level !== false) {
-    info.level = colors.strip(info.level);
+    info.level = colors.strip(info[LEVEL]);
```

Comment:
Looking at this again I believe this is wrong. Will follow-up with a change and a test.